src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -76,6 +76,12 @@ source %{
       case Op_VectorCastHF2F:
       case Op_VectorCastF2HF:
         return UseZvfh;
+      case Op_VectorLoadShuffle:
+      case Op_VectorRearrange:
+        if (vlen < 4) {
```

Comment:
Why the 4? It would be worth adding a comment to explicitly explain why.

Reply:
added, thanks.